src/main/java/io/lettuce/core/KillArgs.java

```diff
@@ -155,6 +157,17 @@ public static KillArgs typeSlave() {
         public static KillArgs user(String username) {
             return new KillArgs().user(username);
         }
+
+        /**
+         * Creates new {@link KillArgs} setting {@literal MAXAGE}.
+         *
+         * @return new {@link KillArgs} with {@literal MAXAGE} set.
+         * @see KillArgs#maxAge(Long)
+         * @since 8.0
```

Comment:
Nit: we could include this change also into the 6.3 line

Reply:
Not sure I fully understand what you mean, Looking at the official code repository, this command will only be available in version redis 8.0.

Reply:
Ah, I see. We're talking two different things. `@since` indicates the library version in which the functionality was added, while you're referring to the server version.